dub.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "sdc",
+    "description": "The snazzy D compiler. Currently only sdfmt (sdc:sdfmt) is available via dub",
+	"targetType": "none",
+	"subPackages": [
+		{
+			"name": "sdfmt",
+			"targetType": "executable",

```

Comment:
The indentation seems to be fuabred, you might want to run this file through prettier.

Reply:
I think this is just an artefact of how github prints it